docs/sql-ref-functions-builtin-aggregate.md

```diff
@@ -19,4 +19,657 @@ license: |
   limitations under the License.
 ---
 
-Aggregate functions
\ No newline at end of file
+Spark SQL provides build-in aggregate functions defined in the dataset API and SQL interface. Aggregate functions
+operate on a group of rows and return a single aggregated value.
+
+<table class="table">
+  <thead>
+    <tr><th style="width:25%">Function</th><th>Argument Type(s)</th><th>Description</th></tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td><b>{any | some | bool_or}</b>(<i>expression</i>)</td>
+      <td>boolean</td>
+      <td>Returns true if at least one value is true.</td>
+    </tr>
+    <tr>
+      <td><b>approx_count_distinct</b>(<i>expression[, relativeSD]</i>)</td>
+      <td>(long, double)</td>
```

Comment:
How about using more SQL-like type names? e.g., `long` -> `bigint` https://github.com/apache/spark/blob/master/sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/FunctionRegistry.scala#L547-L558

Reply:
Also, how about using the same format for optional params? e.g., `(long, double)` -> `(long[, double])`

Reply:
ok